handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -609,6 +610,12 @@ public final SSLEngineResult wrap(
 
                 // Explicit use outboundClosed as we want to drain any bytes that are still present.
                 if (outboundClosed) {
+                    // If the outbound was closed we want to ensure we can produce the alert to the destination buffer.
+                    // This is true even if we not using jdkCompatibilityMode.
+                    if (!isBytesAvailableEnoughForWrap(dst.remaining(), 0, 1)) {
```

Comment:
If the intention here is to check if there is enough space for an alert can we explicitly use `2` instead of `0`? Also if we have `2` in multiple places lets make a define for it.

Reply:
yep thats basically it let me use 2 (can't harm)